src/viking.js

```diff
@@ -1,11 +1,60 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+
+  attack(){
+    return this.strength
+  }
+  receiveDamage(damage){ 
+    this.damage = damage;
```

Comment:
no hace falta el this.damage = damage 😄  con el this.health -= damage es suficiente!!